### ISSUE TITLE
plaintext log: %0A transformed to cleartext <br />

### ISSUE THREAD

**Issue by GoogleCodeExporter** [opencaching/okapi#324]
```
What steps will reproduce the problem?
1. Log a cache on opencaching.pl through OKAPI, comment-format plaintext with 
linebreaks (%0A)
2. The log-entry on the website will contain a <br /> visibly there


What is the expected output? What do you see instead?
It seems that the OKAPI transforms the log to HTML, but the page renders it as 
cleartext

Which revision of OKAPI are you using?
checked with 1055 on oc.pl

Please provide any additional information below.
If I than edit the log, it will be displayed properly formatted in the (html) 
editor and after storing it again also displayed correctly on the page.

Regards
ra_sch
```

Original issue reported on code.google.com by `rsu...@googlemail.com` on 5 Nov 2014 at 6:55

**Comment by following5** [comment]
I have verified the comment_format handling at OCPL. The current status is this:

* submitting plaintext works fine since commit cb7d222b
* submitting HTML is broken: &amp;lt;text&amp;gt; will be eaten when displaying the comment at OCPL sites

So the this issue - #324 - meanwhile is solved, while there is another issue. This can be solved by the same hack like for plaintext:

```
            $formatted_comment = str_replace("&amp;", "&amp;#38;", $formatted_comment);
            $formatted_comment = str_replace("&lt;", "&amp;#60;", $formatted_comment);
            $formatted_comment = str_replace("&gt;", "&amp;#62;", $formatted_comment);
```

**Comment by following5** [comment]
The latter is fixed by commit 40e39687.
